modules/formulize/include/entriesdisplay.php

```diff
@@ -4575,9 +4574,13 @@ function formulize_gatherDataSet($settings=array(), $searches, $sort="", $order=
             // used for trapping the {BLANK} keywords into their own space so they don't interfere with each other, or other filters
             $addToItsOwnORFilter = false;
 
-            if ("creation_uid" == $key OR "entry_id" == $key) {
-                $ele_type = "text";
-            } else {
+            $dataHandler = new formulizeDataHandler(false);
+            $metadataFieldTypes = $dataHandler->metaDataFieldTypes;
```

Comment:
metaDataFieldTypes should be metadataFieldTypes to be consistent with the declaration in the class file